src/pybind/mgr/mgr_module.py

```diff
@@ -583,7 +603,7 @@ class MgrStandbyModule(ceph_module.BaseMgrStandbyModule, MgrModuleLoggingMixin):
     from their active peer), and to configuration settings (read only).
     """
 
-    MODULE_OPTIONS = []  # type: List[Dict[str, Any]]
+    MODULE_OPTIONS:List[Option] = []
```

Comment:
pep8

Reply:
Thanks, updated! Curious: aren't we enforcing this PEP8 rule at mgr or dashboard tox?